src/sql/parts/notebook/notebookActions.ts

```diff
@@ -412,6 +412,7 @@ export class AttachToDropdown extends SelectBox {
 				let attachToConnections = this.values;
 				if (!connection) {
 					this.loadAttachToDropdown(this.model, this.getKernelDisplayName());
```

Comment:
```suggestion
					this.loadAttachToDropdown(this.model, this.getKernelDisplayName(), true);
```

Reply:
Can we just do this instead? Then I'll be happy 😄

Reply:
I wish I would do it , but passing 'true' value makes all the connections reset to 'Select Connection'. It works if the previously selected connection is of type SQL but doesn't work for Big data/Unified connections. Showing 'Select Connection' is driven from sqlops extension based on connection type. I am happy to look at any other changes proposed.